charts/kuberpult/tests/charts_test.go

```diff
@@ -1481,6 +1497,67 @@ db:
 	}
 }
 
+func TestVolumes(t *testing.T) {
+	tcs := []struct {
+		Name            string
+		Values          string
+		VolumeMountSize string
+		target          core.Volume
+		VolumeName      string
```

Comment:
`VolumeName` in all cases is "repository". doesn't have to be inside the testcases struct. Can be a constant value.